### PR TITLE
DPE-2617 autotune production profile

### PR DESCRIPTION
**Pull request by paulomach**
Port of pg-k8s [pr#260](https://github.com/canonical/postgresql-k8s-operator/pull/260)

### CODE REVIEW

**Review by taurus-forever — APPROVED** [review]
Please add basic test here to make Codecov happy. Tnx!

File: lib/charms/postgresql_k8s/v0/postgresql.py
Comment (dragomirp) [inline lib/charms/postgresql_k8s/v0/postgresql.py:1]: We should increment the lib version. I've created https://github.com/canonical/postgresql-k8s-operator/pull/264 for k8s

**Review by dragomirp — APPROVED** [review]

File: lib/charms/postgresql_k8s/v0/postgresql.py
Comment (taurus-forever) [reply lib/charms/postgresql_k8s/v0/postgresql.py:1]: Ufff..... we need an automated protection here on CI/CD. Good ideas are welcome!

**Review by marceloneppel — APPROVED** [review]
Thanks Paulo! LGTM.

**Comment by paulomach** [comment]
I'm merging without tests end because the only change was a `libpatch` bump to be consistent with the k8s lib. (previous tests run was successful)

**Pull request merged**